extensions/inkeep/src/services/inkeep.ts

```diff
@@ -0,0 +1,426 @@
+import axios from "axios";
+import { getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  apiKey: string;
+  aiApiBaseUrl: string;
+  model: string;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`